src/utilities/notifications.ts

```diff
@@ -1,5 +1,21 @@
+import { store as noticesStore } from '@wordpress/notices';
+import { dispatch, select } from '@wordpress/data';
 import type { UserState } from '@/store/awareness-store';
+import { store as settingsStore, Setting } from '@/store/settings-store';
 
+export enum NotificationType {
+	PostRestored = 'remote-user-post-restored',
```

Comment:
I'm going to be making a follow up PR to actually enable these notifications, so I'll tweak the enums then. It's fine if they aren't 100% accurate because this code is not being used at the moment and will be changed by me soon.